scania-digital-design-system/tegel
# fix(dropdown): reset scroll

## **Describe pull-request**  
Reset the scroll height of dropdown component when triggering the reset method.

## **Issue Linking:**  
- **Jira:** [CDEP-3371-reset-dropdown-scroll](https://tegel.atlassian.net/browse/CDEP-3371)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to story file for Dropdown
2. Add a reset button  `<button onclick="dropdown.reset()">Reset</button>`
3. Scroll in the dropdown menu and select an option
4. Press reset button and see that the dropdown is scrolled back up/down.
5. Change `Open Direction` and repeat step 3 & 4.

## **Suggested test steps**
